File: examples/new_robot.rs
use std::f64::consts::PI;

use libjaka::JakaMini2;
use robot_behavior::{RobotResult, behavior::*};

fn main() -> RobotResult<()> {
    let mut robot = JakaMini2::new("10.5.5.100").with_cartesian_vel(1.0);
    robot.enable()?;
    robot.stop()?;

    // 以受限的笛卡尔速度移动到法兰目标位姿（平移单位：米，姿态单位：弧度）。
    robot.move_to::<FlangeSpace>(Pose::Euler([0.3, 0.0, 0.03], [PI, 0.0, PI]))?;

    Ok(())
}

File: scripts/new_robot.py
import libjaka.libjaka


pose = libjaka.Pose.Euler([1,2,3],[2,1,3])
print(pose.quat())

robot = libjaka.JakaRobot("10.5.5.100")

robot.move_joint([0, 0, 0, 0, 0, 0], 1)
robot.move_cartesian_int(pose, 1)